modules/host/persist.go

```diff
@@ -95,10 +95,7 @@ func (h *Host) establishDefaults() error {
 		return err
 	}
 	h.secretKey = sk
-	h.publicKey = types.SiaPublicKey{
-		Algorithm: types.SignatureEd25519,
-		Key:       pk[:],
-	}
+	h.publicKey = types.Ed25519PublicKey(pk)
```

Comment:
Interesting. I guess this is good, though the types package now has a crapload of functions.

But... probably not a bad thing if they are all well documented. It is the best starting place for learning the Sia codebase.